bpy_speckle/__init__.py

```
@@ -36,6 +26,14 @@
 # States
 from .states.speckle_state import register as register_speckle_state, unregister as unregister_speckle_state
 
+import bpy
+from .ui import icons
+import json
+
+# Ensure dependencies
+from .installer import ensure_dependencies
+ensure_dependencies(f"Blender {bpy.app.version[0]}.{bpy.app.version[1]}")
```

What was the reason for moving this segment,
There is a ruff error generated by doing this, but actually its essential that this line is executed before we attempt to import any dependent code (like specklepy)

I would suggest moving this back to the top of the class, and supressing any ruff error.

---

oh, i see. i moved it because of the ruff warning. will suppress the error then.